File: Validate-BST/validate-bst.py
# Python Solution
# BST input class

class BST:
  def __init__(self, value):
    self.value = value
    self.left = None
    self.right = None


# O(n) time | O(d) space
def validateBst(tree):
  return validateBstHelper(tree, float("-inf"), float("inf"))

  

File: Validate-BST/validate-bst.js
// JavaScript Solution
// BST input class

class BST {
	constructor(value) {
		this.value = value;
		this.left = null;
		this.right = null;
	}
}

// O(n) time | O(d) space
function validateBst(tree) {
	return validateBstHelper(tree, -Infinity, Infinity);
}
